servers/parser-graphql-wrapper/src/marticle/marticleParser.ts

```diff
@@ -526,6 +559,9 @@ function listTransformer(
   output: Array<MarticleElement | NumberedListElement | ListElement>;
   aggFrom: number | null;
 } {
+  if (!node.hasChildNodes) {
```

Comment:
This was the fix for the commented on block above, however when adding this, the main marticle output then gets empty arrays added to it.